build.sbt

```diff
@@ -680,6 +680,12 @@ lazy val zio2 = (project in file("zio-2"))
     libraryDependencies += "dev.zio" %% "zio-json" % "0.7.3",
     libraryDependencies += "dev.zio" %% "zio-test" % zioVersion % Test,
     libraryDependencies += "dev.zio" %% "zio-test-sbt" % zioVersion % Test,
+    libraryDependencies += "dev.zio" %% "zio-logging" % "2.1.10",
+    libraryDependencies += "dev.zio" %% "zio-logging-slf4j2" % "2.1.10",
+    libraryDependencies += "ch.qos.logback" % "logback-classic" % "1.5.6",
+//    libraryDependencies += "dev.zio" %% "zio-logging-slf4j2-bridge" % "2.1.10",
```

Comment:
let's remove commented out stuff like this

Reply:
This dependency is used in an example in the article, but it can't work with the other dependencies. I was under the impression that I should keep it for consistency with the article. Should I remove it?

Reply:
Got it - let's add a line indicating this dependency is added during the article